cmd/main.go

```
@@ -162,6 +162,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&controller.BucketLifecyclePolicyReconciler{
+		Client:   mgr.GetClient(),
+		Log:      ctrl.Log.WithName("controller").WithName("BucketLifecyclePolicy"),
+		Scheme:   mgr.GetScheme(),
+		Recorder: mgr.GetEventRecorderFor("bucketlifecyclepolicy-controller"),
```

golangci-lint (2.12.2)

[error] SA1019: mgr.GetEventRecorderFor is deprecated: this uses the old events API and will be removed in a future release. Please use GetEventRecorder instead. 

(staticcheck)